Common/File/FileUtil.cpp

```diff
@@ -231,43 +290,44 @@ bool Exists(const std::string &filename) {
 	return true;
 #else
 	struct stat file_info;
-	return stat(fn.c_str(), &file_info) == 0;
+	return stat(path.c_str(), &file_info) == 0;
 #endif
 }
 
-bool Exists(const Path &path) {
-	// Temporary forward.
-	return Exists(path.ToString());
-}
-
 // Returns true if filename exists and is a directory
 bool IsDirectory(const Path &filename) {
 	switch (filename.Type()) {
 	case PathType::NATIVE:
 		break; // OK
+	case PathType::CONTENT_URI:
+	{
+		FileInfo info;
+		if (!Android_GetFileInfo(filename.ToString(), &info)) {
+			return false;
+		}
+		return info.isDirectory;
```

Comment:
`exists &&`?  I think that's used in another place or two.

-[Unknown]

Reply:
Yup